package.json

```diff
@@ -54,6 +54,7 @@
     "@types/node": "^15.3.1",
     "@types/saslprep": "^1.0.0",
     "@types/semver": "^7.3.4",
+    "@types/whatwg-url": "^8.2.1",
```

Comment:
what's this one for?

Reply:
It was to fix this: https://github.com/mongodb-js/mongodb-connection-string-url/issues/2
But its been merged in one of my other PRs so I'll rebase :)